governance/polls/Proposal - Adjust the Dust Parameter

```diff
@@ -0,0 +1,37 @@
+---
+title: Adjust the Dust Parameter - August 10, 2020
+summary: Signal your support for increasing the Dust Parameter from 20 DAI to 100 DAI
```

Comment:
Can we have 'for all vault types.' In here as well, please?

Reply:
👍